src/TripRepo.js

```diff
@@ -21,6 +21,24 @@ class TripRepo {
     let end = new Date( year + 1, 0, 0);
     return new TripRepo(this.data.filter(trip => time.isBetween(beg, trip.date, end)), null, false);
   }
+
+  getUpcomingFolio(date) {
+    return new TripRepo(this.data.filter(trip => time.isBefore(date, trip.date)), null, false);
```

Comment:
Could there be a way to refactor the trip repo so you don't have to put null and false every time you create a TripRepo?